### PR TITLE
add script to compare git and debian repo

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: scripts/compare_deb_repo.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+import glob
+import os
+import requests
+from debian.deb822 import Packages
+from debian.changelog import Changelog
+
+DISTS = [os.path.basename(d) for d in glob.glob('debian/*') if d != 'debian/buildscripts']
```
Comment (ekohl) [inline scripts/compare_deb_repo.py:9]: I like using `pathlib.Path`
```suggestion
DISTS = [p.name for p in Path('debian').glob('*') if p.name != 'buildscripts']
```

**Review by ekohl — APPROVED** [review]
This looks smarter than what I wrote in https://github.com/ekohl/theforeman-plumbing/blob/master/packages/deb/application.py (and I never finished the git part @ https://github.com/ekohl/theforeman-plumbing/blob/master/packages/git/build_cache).

**Comment by evgeni** [comment]
@ekohl I went ahead and fully migrated to pathlib (also the glob parts), mind having a look before I merge?

File: scripts/compare_deb_repo.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            continue
+        repo_packages.add((source, version))
+    return repo_packages
+
+
+def get_git_packages(dist):
+    if dist == 'plugins':
+        package_folders = Path('plugins/').glob('*/debian')
+    else:
+        package_folders = chain(Path('debian/').glob(f'{dist}/*'), Path('dependencies/').glob(f'{dist}/*'))
```
Comment (ekohl) [inline scripts/compare_deb_repo.py:29]: I normally don't use a trailing slash for directories but it doesn't make a difference.

**Review by ekohl — APPROVED** [review]

**Pull request merged**